.github/workflows/trigger-gitlab.yml

```diff
@@ -0,0 +1,59 @@
+# inspired by rhinstaller/anaconda
+
+name: Trigger GitLab CI
+
+on:
+  pull_request:
+    branches:
+      - "*"
```

Comment:
Why on pull requests? Afaik we barely check the results on main for the other projects.

Reply:
Idk, I guess for consistency sake? We have it like that in all other repos. I know nobody checks the results but I just want to have them in case we ever decide we want them... It takes a few minutes and spins one aws machine if I don't count the github runner...

Reply:
I've changed it to run just on `main`